### PR TITLE
Install postgresql-devel

### PR DESCRIPTION
**Pull request by jgrund**
Ref:
https://github.com/whamcloud/integrated-manager-for-lustre/issues/2048

Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: copr-rust/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,9 @@
 FROM imlteam/copr
 
 ARG toolchain=stable
-RUN yum install -y gcc \
+RUN yum install -y gcc https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm \
   && yum clean all \
+  && yum install -y postgresql96-devel \
```
Comment (ip1981) [inline copr-rust/Dockerfile:6]:
```suggestion
  && yum install -y postgresql96-devel \
  && yum clean all \
```

Comment (jgrund) [reply copr-rust/Dockerfile:6]: It shouldn't matter. It's basically unsafe to install any packages in an automated way due to the way epel metadata shifts.

Comment (jgrund) [reply copr-rust/Dockerfile:6]: And I should add that due to epel we need the clean prior to the install as well.

Comment (ip1981) [reply copr-rust/Dockerfile:6]: I mean you might want to remove unneeded files from the image after yum finished.

Comment (jgrund) [reply copr-rust/Dockerfile:6]: We can add it afterwards as well, but it still needs to be before.

**Pull request merged**